tests/test_ticking.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+import time
+import mock
+
+from freezegun import freeze_time
+from tests import utils
+
+
+@utils.cpython_only
+def test_ticking_datetime():
+    with freeze_time("Jan 14th, 2012", tick=True):
+        assert datetime.datetime.now() > datetime.datetime(2012, 1, 14)
```

Comment:
insert a tiny `sleep()` here of `0.1` seconds, or even `0.01` seconds - some system clocks don't have microsecond resolution which means the test could intermittently fail there.